conf.d/main

```diff
@@ -17,7 +17,7 @@ sed -i 's|^\(crl=\).*|\1"crl.jail/crl.pem"|' $EASY_RSA/openssl-easyrsa.cnf
 # webserver related
 mkdir -p /var/www/openvpn/logs
 chown www-data:www-data /var/www/openvpn/logs
-lighty-disable-mod ssl
+ln -s /etc/lighttpd/conf-{available,enabled}/50-tklcp.conf
```

Comment:
Obviously what you did is fine, but FYI this could be `lighty-enable-mod tklcp` instead.

Reply:
This tripped me up, didn't realize lighty-enable-mod refered to conf/sites, cheers :p